osu.Game.Tournament/Screens/Drawings/DrawingsScreen.cs

```diff
@@ -36,12 +36,12 @@ public class DrawingsScreen : TournamentScreen
 
         private Task writeOp;
 
-        private Storage storage;
+        private TournamentStorage storage;
 
         public ITeamList TeamList;
 
         [BackgroundDependencyLoader]
-        private void load(TextureStore textures, Storage storage)
+        private void load(TextureStore textures, TournamentStorage storage)
```

Comment:
This one is confusing too, but I can understand why this would be required more (due to caching in `TournamentGameBase`, I presume). You can `CacheAs<Storage>`, fwiw.